terraform/product/variables.tf

```diff
@@ -0,0 +1,41 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+variable "model" {
+  description = "Reference to the Juju model to deploy the github-runner and github-runner-image-builder operators."
+  type        = string
+}
+
+variable "github_runner_image_builder" {
+  type = object({
+    app_name    = optional(string, "github-runner-image-builder")
+    channel     = optional(string, "latest/edge")
```

Comment:
Should the product terraform plan be on edge?
I would thought stable would be the default choice.